lib/commands/instrument.js

```diff
@@ -111,6 +117,7 @@ exports.handler = function (argv) {
     preserveComments: argv.preserveComments,
     include: argv.include,
     exclude: argv.exclude,
+    excludeNodeModules: argv.excludeNodeModules,
```

Comment:
can we just spread the argv object here instead of having to add each option manually?

```js
const nyc = new NYC({ ...argv })
```

Reply:
Actually the main command and all other sub-commands just pass argv directly to nyc.  I'm going to review the call path to make sure this doesn't unexpectedly introduce any options that were previously unused but now get used.

Reply:
I only suggested the spread in case we needed to override certain options for whatever reason, but yeah that sounds good!

Reply:
Looks like I'm going to be doing a follow-up to add a bunch of flags to the `nyc instrument --help` listing.  Nothing harmful but this will pass a few options to `nyc instrument` if they're set via configuration (cache, cache-dir are two examples).

I think I'm going to do a followup which will create a `lib/commands/shared-flags.js`, this will export functions that add groups of flags - for example one will add the test-exclude options, one will add the istanbul-lib-instrument options, etc.

Reply:
Additional note: we can't use object spread yet, not until we drop node.js 8.